mage_ai/orchestration/pipeline_scheduler.py

```diff
@@ -142,6 +146,7 @@ def run_block(pipeline_run_id, block_run_id, variables, tags):
         update_status=False,
         on_complete=pipeline_scheduler.on_block_complete,
         on_failure=pipeline_scheduler.on_block_failure,
+        on_start=pipeline_scheduler.on_block_start,
```

Comment:
Let's set the start time right after we update the block_run status to RUNNING. https://github.com/mage-ai/mage-ai/pull/1010/files#diff-69fb57191733e0b7e80fa40cd25c3f687fefbf0bddcc87724d7a7a85337739d4R104